src/main/java/mekanism/client/gui/item/GuiPortableTeleporter.java

```diff
@@ -117,6 +121,11 @@ public double getLevel() {
             }
             updateButtons();
         }));
+        func_230480_a_(new GuiSlot(SlotType.NORMAL, this, 131, 120));
```

Comment:
This should have a `.setRenderAboveSlots()` on the end of the slot creation like the logistical sorter does. (If it doesn't in your code base just pull in/rebase onto the latest 1.16 branch to see what I mean)

Reply:
done.